public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+        incrementing the instruction address
+        would make it overflow.
+        The carry would add one to the
+        operation code, changing it to the next one in the instruction set:
+
+
+If incrementing the address span overflowed into the opcode span, then the bit order between them is established:
+
+            CCCXAAANNN
+        MSB <----------> LSB
+
+If the index register bit `(X)` is indeed between the two and turned on, then overflowing the `(A)` span will carry through `(X)` into the `(C)` span, incrementing it by one. The result:
+
+        a jump instruction.
+        Sure enough, the next program instruction was
+        in address location zero,
+        and the program went happily on its way.
+
+### The Unpleasant Truth
+
+All this is possible in theory, but we still don't know where the `JUMP` instruction takes its operand from. We know that the operand's value must be `0` and it should be ready. The address span `(A)` indeed contains `0`, but nowhere in the description of RPC-4000's architecture, does Nather mention an option of the data address field doubling as an instruction address. The `JUMP` instruction could take its value from some register, but that `0` would have to be stored there beforehand, providing a screaming clue that some operation was being set up.
+
+Not only did I gloss over this crucial step for 30 years - I also ignored the absence from this part of the story, of the unique addressing mode described above. The one that included the next instruction address in every instruction(!). Was that part of affected in the overflow? Was there any relationship between the `JUMP` opcode and the `(A)` or `(N)` fields? And what about that `(X)` bit? Its alleged location, between two bitfields, just seemed... wrong.
+
+After demoting myself from 'computer guy' to 'A guy who likes to think of himself as such', I took the basic step required to understand Mel's hack: looking up the [RPC-4000 manual](http://www.bitsavers.org/pdf/royalPrecision/RPC-4000/RPC-4000_Programming_Manual.pdf). It didn't take much browsing to hit a figure that dispelled all my doubts.
+
+![RPC 4000 Instruction format](https://res.cloudinary.com/dcajl1s6a/image/upload/v1654892829/mels-hack/RPC_4000_Instruction_ypjaii.png)
+
+Quite simply, the hack, as described in Ed Nather's account, is impossible on the RPC-4000. The opcode `(C)` field, supposedly modified by the overflow, is in the least significant bits of the instruction. In the terms used above:
+
+             XNNNAAACCC
+        MSB <----------> LSB
+
+
+Thus, any overflow (which progresses toward the MSB) in the bits above the opcode, would not affect the latter. Furthermore, opcode `0` was not "A Jump instruction", but rather one of two operations - `HLT` or `SNS` - the specifics of which are beyond the scope of this analysis. Thus, even a different bit arrangement would not have redeemed the described hack.
+
+
+
+### Reconstructing the Hack...
```

Comment:
Remove ellipsis

Reply:
Seems like this paragraph can be merged; it's a single paragraph with its own title.